client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -271,15 +273,19 @@ export function WPNonProductLineItem( {
 			{ hasDeleteButton && removeProductFromCart && formStatus === FormStatus.READY && (
 				<>
 					<DeleteButton
-						coupon={ coupon }
 						className="checkout-line-item__remove-product"
-						buttonType="borderless"
+						buttonType={ isDeleteButtonText ? 'text-button' : 'borderless' }
 						disabled={ isDisabled }
+						isDeleteButtonText={ isDeleteButtonText }
 						onClick={ () => {
 							setIsModalVisible( true );
 						} }
 					>
-						<DeleteIcon uniqueID={ deleteButtonId } product={ label } />
+						{ isDeleteButtonText ? (
+							translate( 'Remove from cart' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **32** times:
`translate( 'Remove from Cart' )` *ES Score: 9*